src/04-accordion/style.scss

```diff
@@ -62,114 +71,58 @@ SASS modifications to conform GovUK frontend to MoJ block styling
         margin-left: govuk-px-to-rem(22);
     }
 
-    &__section {
-        &:active {
-            button:active {
-                top: 0;
+    &__section-content {
+        // Decrease padding causes by NHS styles around accordion content
+        padding-top: 0;
+
+        // Indent body text in accordion section content to align with title
+        .govuk-body {
+			margin-left: 16px;
+			margin-left: govuk-px-to-rem(16);
+            padding-left: 30px;
+            padding-left: govuk-px-to-rem(30);
+        }
+    }
+
+    // When accordion seciton is open.
+    &__section--expanded {
+        .govuk-accordion {
+            &__section-heading {
+                &:before {
+                    display: block;
+                    width: 0;
+                    height: 0;
+                    border-style: solid;
+                    border-color: transparent;
+                    border-top-color: transparent;
+                    -webkit-clip-path: polygon(0% 0%, 50% 100%, 100% 0%);
+                    clip-path: polygon(0% 0%, 50% 100%, 100% 0%);
+                    border-width: 12.124px 7px 0 7px;
+                    border-top-color: inherit;
+                }
             }
         }
+
     }
 
     // Remove "mustaches" underline caused by NHS styles conflicting with Gov UK
     &__section-button,
     &__open-all {
         box-shadow: none;
         border-radius: 0;
+
+        &:active {
+            top: 0;
+        }
     }
 
-    // Change position of open/close icon to far left side
+    // Remove +/- that comes with accordion (replaced with arrows).
     &__icon {
-        right: 0;
-        left: -30px;
-        width: 16px;
-        height: 16px;
-        margin-top: -7px;
-        margin-top: govuk-px-to-rem(-7);
-
-        &:after {
-            display: none;
-        }
+        display: none;
     }
 
-    // Add a bottom border to accordion sections (on last one)
+    // Add a bottom border to last accordion section
     &:last-child {
         border-bottom: 1px solid $mojblocks-color-light-grey;
     }
 }
-
-// Replace Gov plus symbol with arrow symbol (when open)
-.js-enabled {
```

Comment:
Do we not need js-enabled any more?

Reply:
noope, nothing depends on that class now.